WTWSMS/decisions/VIET_Eastern_religions_decisions.txt

```diff
@@ -31,7 +32,7 @@ decisions = {
 					trait = kind
 				}
 			}
-			NOR = {
+			NOT = {
```

Comment:
@loup99 

Isn't that what "NOT" does? If anything in a "NOT" zone is true, the entire thing is rejected. Is "NOR" the opposite?